front/src/applications/stdcmV2/components/StdcmResults/StdcmSimulationBanner.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { CheckCircle } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import type { StdcmSimulation } from 'applications/stdcmV2/types';
+import { formatDateToString } from 'utils/date';
+
+type StdcmSimulationBannerProps = {
+  simulationsList: StdcmSimulation[];
+  retainedSimulationIndex: number;
+  selectedSimulationIndex: number;
+  setSelectedSimulationIndex: React.Dispatch<React.SetStateAction<number>>;
+  onInteractWithOutputs: () => void;
+};
+
+const StdcmSimulationBanner = ({
+  simulationsList,
+  retainedSimulationIndex,
+  selectedSimulationIndex,
+  onInteractWithOutputs,
+  setSelectedSimulationIndex,
+}: StdcmSimulationBannerProps) => {
+  const { t } = useTranslation('stdcm');
+  const shorterYear = true;
+
+  const onSelectSimulation = (index: number) => {
+    if (retainedSimulationIndex === -1) {
+      setSelectedSimulationIndex(index);
+      onInteractWithOutputs();
+    }
+  };
```

Comment:
Instead of updating PathSteps multiple times in all origin/vias/dest, we can dispatch here

```suggestion
  const onSelectSimulation = (index: number) => {
    if (retainedSimulationIndex === -1) {
      setSelectedSimulationIndex(index);
      onInteractWithOutputs();
      dispatch(updatePathSteps({ pathSteps: simulationsList[index].inputs.pathSteps! }));
    }
  };
```

Reply:
You raise a very good point. However, I believe there's more to consider. We also need to push `rollingStockID`, `originDate`, and `originTime`. To handle all of these together efficiently, it's better to manage them in one place.

That's why I added a new action, resetStdcmConfigWithData, which I call from the StdcmView2 component. This approach allows us to centralize the logic, as explained in my previous comment. It simplifies the state management and keeps our code cleaner and more maintainable.

Reply:
Even better, very nice ! Just think the name is strange "resetting but with data". Maybe it could be `updateStdcmConfigWithData` or `loadSelectedSimulationInputs`. What do you think ?

Reply:
Done